Added obsidian-file-chucker to the list

kenlim opened the pull request:
Added obsidian-file-chucker to the list

# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/kenlim/file-chucker-plugin

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [X]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [X] My GitHub release contains all required files
  - [X] `main.js`
  - [X] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [X] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [X] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [X] My README.md describes the plugin's purpose and provides clear usage instructions.
- [X] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [X] I have added a license in the LICENSE file.
- [X] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

liamcain commented:
- [await app.vault.getAbstractFileByPath](https://github.com/kenlim/obsidian-file-chucker/blob/a0495cb7106f60ee658d43ba9d82b7ad67381fe9/src/main.ts#L157) this function isn't async. You shouldn't need the await
- [editorCallback](https://github.com/kenlim/obsidian-file-chucker/blob/a0495cb7106f60ee658d43ba9d82b7ad67381fe9/src/main.ts#L37) editorCallback is only called if the file is in edit mode. If a user prefers preview mode, this command won't be available. I suggest using [checkCallback](https://github.com/obsidianmd/obsidian-api/blob/9bb30633586c6649474b2affdf5b87efdd86498f/obsidian.d.ts#L505) and checking for `this.app.workspace.getActiveViewOfType(MarkdownView)`

liamcain commented on community-plugins.json:
```diff
@@ -5788,5 +5788,12 @@
+    },
+    {
+        "id": "obsidian-file-chucker",
+        "name": "Obsidian file chucker",
```
We're asking plugins to avoid using the word "Obsidian" in their plugin name to avoid desaturating the brand name and reserve the naming for first-party plugins (see: Obsidian Publish, Obsidian Sync).

kenlim replied:
Have changed this to just "file-chucker"

liamcain commented on community-plugins.json:
```diff
@@ -5788,5 +5788,12 @@
         "branch": "main"
+    },
+    {
+        "id": "obsidian-file-chucker",
```
Similarly, you don't need to include the word "obsidian" in your ID.

kenlim replied:
it's just "file-chucker" now.

kenlim commented:
Have removed await to getAbstractFileByPath.
Have also changed to the editorCallback, so this plugin now works in Preview mode!
Also changed the plugin name and repo name to match
